_includes/blog_modal.html

```diff
@@ -0,0 +1,24 @@
+<div class="container">
+    <div class="modal fade" id="blog-modal" role="dialog">
+        <div class="blog-modal modal-dialog modal-lg">
+            <div class="modal-content">
+                <div class="modal-body">
+                    <img data-toggle="modal" data-target="#blog-modal" src="{{ site.baseurl }}/assets/images/{{include.blog-image}}" width="100%">
+                </div>
+                <div class="modal-footer">
+                    <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+
+
+<script src="https://cdn.jsdelivr.net/npm/js-image-zoom/js-image-zoom.min.js"></script>
+<script>
+    var options = {
+        width: 400, // required
+        zoomPosition:"original"
+    };
+    new ImageZoom(document.getElementById("img-container"), options);
+</script>
```

Comment:
nit - You can use 2 spaces here too, and put a space between `zoomPosition:` and `"original"`.